src/foraging_gui/Foraging.py

```diff
@@ -155,7 +168,17 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
         self._WaterVolumnManage2()
         self._LickSta()
         self._InitializeMotorStage()
-        self._GetPositions()
+
+        # Initialize Allen Institute stage widget if Newscale stage was not found
+        if not hasattr(self, 'current_stage'):
```

Comment:
Lets actually use `if self.Settings['newscale_serial_num_box{}'.format(self.box_number)] == '':` as the check. Since current_stage could be missing if we intend to use a newscale stage, and it didn't connect for some reason.